app/client/src/components/pages/Community.Tabs.IdentifiedIssues.js

```diff
@@ -392,7 +392,7 @@ function IdentifiedIssues() {
       );
 
       if (mappedParameterName) {
-        parameters.push(mappedParameterName);
+        parameters.push(mappedParameterName.label);
```

Comment:
```suggestion
        parameters.push(mappedParameter.label);
```

Can you change the `mappedParameterName` variable (line 389) to `mappedParameter` and the `getMappedParameterName` function (line 149) to `getMappedParameter`?  This will just make the names more accurate.

Reply:
Good point!